u8191108-lab1/comment.php

```diff
@@ -0,0 +1,65 @@
+<div>
+    <form action="" method="post" style="display: block;">
+        <p>Comments by users created after date:</p>
+        <input type="date" name="date">
+        <input type="submit" name="Filter">
+    </form>
+</div>
+
+<?php
+require 'user.php';
+
+class Comment
+{
+    private User $user;
+    private string $text;
+
+    function __construct(User $user, string $text) {
```

Comment:
Можно использовать короткую форму объявления свойств класса
```suggestion
    public function __construct(private User $user, private string $text)
    {
    }
```